testing/test/init-activate_multi_jobs_single_gpu/main.cpp

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
This empty line at the beginning seems unnecessary. Perhaps also missing a license?

Reply:
I have updated all the the `.cpp` and `.cu` files in the PR to have licenses.